django_switchuser/urls.py

```diff
@@ -2,7 +2,12 @@
 
 from . import views
 
-urlpatterns = patterns("",
+patterns = [
     url(r"^$", views.su_login, name="su-login"),
     url(r"^logout$", views.su_logout, name="su-logout"),
-)
+]
+
+try:
+    urlpatterns = patterns("", *patterns)
+except NameError:
```

Comment:
You're totally right, I also missed the import for patterns as well.  If we try to use 'patterns' after the import fails, I think we end up with `NameError: name 'patterns' is not defined`. 


I've updated it just to explicitly check for the django version instead, hopefully that's a bit clearer!